packages/crawler/src/page-processors/page-processor-base.ts

```diff
@@ -125,7 +148,11 @@ export abstract class PageProcessorBase implements PageProcessor {
         await this.blobStore.setValue(`${scanData.id}.data`, scanData);
     }
 
-    protected async logPageError(request: Apify.Request, error: Error): Promise<void> {
+    protected async logBrowserFailure(request: Apify.Request, browserError: BrowserError): Promise<void> {
```

Comment:
we need to have the same function, only logError, the file should be the error trace (more details), error type column would be error type, and errorDescription column will be error.message

Reply:
added error stack to browser error